apps/google-analytics-4/frontend/src/components/main-app/ChartContent/ChartContent.tsx

```diff
@@ -21,16 +24,24 @@ const ChartContent = (props: Props) => {
     });
   };
 
-  return (
-    <>
+  const renderChartContent = () => {
+    if (error) {
+      return <Note body={error?.message || DEFAULT_ERR_MSG} variant="negative" />;
+    } else if (pageViewData.rows && !pageViewData.rows.length) {
+      return <Note body={EMPTY_DATA_MSG} variant="warning" />;
+    }
```

Comment:
Since these two if statements aren't related to each other, I feel that separating them would help make this easier to skim:
```suggestion
    if (error) {
      return <Note body={error?.message || DEFAULT_ERR_MSG} variant="negative" />;
    }
    
    if (pageViewData.rows && !pageViewData.rows.length) {
      return <Note body={EMPTY_DATA_MSG} variant="warning" />;
    }
```

Reply:
Great suggestion, will adjust 👍 !